src/sdk/constants.ts

```diff
@@ -8,3 +8,4 @@ dkRj2Je2kag9b3FMxskv1npNSrPVcSc5lGNYlnZnfxIAnCknOC118JjitlrpT6wd
 8wIDAQAB
 -----END PUBLIC KEY-----
 `;
+export const KERNEL_APP_ID = 0;
```

Comment:
How is this value supposed to be set when using the SDK?

Reply:
it's not supposed to be set when using the SDK, we will set it to the app id of the official ubiquibot so plugins can use it while matching for bot comments or similar stuff

Reply:
Got it. Shouldn't we set the value now instead of 0?

Reply:
@Whilefoo can you confirm that `APP_ID` can be used to target but comments? As you may be aware, we have that open pool request by key ring which claims that this doesn't work.

Reply:
then we will export both BOT_USER_ID and APP_ID if the plugin developer needs them